cli/lib/kontena/cli/stacks/deploy_command.rb

```diff
@@ -21,7 +23,9 @@ def execute
       spinner "Waiting for deployment to start" do
         wait_for_deployment_to_start(deployment)
```

Comment:
I don't see why we would want to wait for the deployment to start if we're not gonna wait for it to finish either. So I'd wrap this also within  `if !async`